recipes/ssh-crypt/meta.yaml

```
@@ -0,0 +1,40 @@
+{% set name = "ssh-crypt" %}
+{% set version = "1.1.3" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/ssh-crypt-{{ version }}.tar.gz
+  sha256: 69cd2e52dc33c6e92b16247ca50a66325b868899125a3d9a0c4c7b88e9283a33
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - paramiko
+    - cryptography
+
+test:
+  imports:
+    - ssh_crypt
+  requires:
+    - paramiko
+    - cryptography
+
+about:
+  home: https://pypi.org/project/ssh-crypt/
+  summary: ssh-crypt is a tool to encrypt/decrypt data using your ssh key from ssh-agent
+  license: BSD-1-Clause
```

Looks like it's actually:
```suggestion
  license: BSD-3-Clause
```

---

done